torch_em/data/datasets/livecell.py

```diff
@@ -143,19 +145,23 @@ def _livecell_segmentation_loader(
                                               label_dtype=label_dtype,
                                               transform=transform,
                                               n_samples=n_samples)
-    return torch_em.segmentation.get_data_loader(ds, batch_size, **loader_kwargs)
+
```

Comment:
This change is not necessary. `get_data_loader` already does the right thing: https://github.com/constantinpape/torch-em/blob/main/torch_em/segmentation.py#L302